### DISCUSSION TITLE
v3.0.4

### DISCUSSION THREAD

**Discussion by DTTerastar** [ESPresense/ESPresense#644] [Announcements]
## What's Changed
* Fixes iPhone tracking not home #641, #640, #550, #460 by @DTTerastar in https://github.com/ESPresense/ESPresense/pull/643

**Full Changelog**: https://github.com/ESPresense/ESPresense/compare/v3.0.3...v3.0.4

<hr /><em>This discussion was created from the release <a href='https://github.com/ESPresense/ESPresense/releases/tag/v3.0.4'>v3.0.4</a>.</em>

**Comment by janssenjones** [comment]
This release has so far been solid with iPhone and Apple Watch detection.  Great work!  
One unexpected behavior (maybe by design) is that it seems the espresence/ui/devices#/fingerprints page is no longer using the filters.  In prior releases, what went to MQTT and what went to this page matched, but now, my UI page is showing a full firehose (I have maybe 40+ devices in my house (Tiles, air tags, Mi ATC devices, etc) , but am only tracking 10 of them)... I can live it with and look to MQTT if that's the new design, but before, I kept the fingerprints page up everywhere for easier troubleshooting...

**Reply by 51av0sh** [reply]
Strange. It's filtering fine for me.

**Reply by DTTerastar** [reply]
I need to have a "show all" option so we can show all for debugging.  That would have helped find the 3.0.4 issue...
